python-bindings/CMakeLists.txt

```diff
@@ -1,8 +1,3 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.14 FATAL_ERROR)
```

Comment:
What is the thought behind dropping the cmake minimum check?

Reply:
Just that we have it already as the very first line in the top CMakeLists.txt.